src/app/pages/wallets/wallet-details/pages-wallet-details.types.ts

```diff
@@ -0,0 +1,18 @@
+export interface WalletsDetails {
+  id: number;
+  name: string;
+  transaction: WalletTransactionDetails[];
+}
+
+export interface WalletTransactionDetails {
+  id: number;
+  date: string;
+  description: string;
+  amount: number;
+}
+
+export enum WalletTransactionType {
```

Comment:
I will later adjust linter config for that but for enums lets go with that convention:
- Keys starting with capital letter so for example `Incomes`
- Values uppercase letters with underscore like `ALL_TRANSACTIONS`